lib/components/radio_row/radio_row.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:tarka_ui/styles/theme.dart';
+
+import '../radio_button/radio_button.dart';
+
+class TUIRadioRow extends StatelessWidget {
+  final String title;
+  final String description;
+  final VoidCallback? onPressed;
+  final bool? isSelected;
+  final bool backgroundDark;
+
+  const TUIRadioRow({
+    super.key,
+    required this.title,
+    this.description = "",
+    this.onPressed,
+    this.isSelected,
+    this.backgroundDark = false,
+  });
+
+  TUIRadioButton getRadioButton() {
+    if (isSelected == null) {
+      return TUIRadioButton(
+        onPressed: () {},
+      );
+    } else {
+      return TUIRadioButton(
+        onPressed: () {},
+        isSelected: isSelected ?? false,
+      );
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = TUITheme.of(context);
+    final showDescription = description.isNotEmpty;
+
+    return Container(
+      padding: const EdgeInsets.all(8.0),
+      decoration: BoxDecoration(
+        color: backgroundDark ? theme.colors.background : Colors.transparent,
+        borderRadius: const BorderRadius.all(
+          Radius.circular(16.0),
+        ),
+      ),
+      child: Row(
+        mainAxisSize: MainAxisSize.max,
+        crossAxisAlignment: getRowCrossAxisAlignment(description),
+        children: [
+          getRadioButton(),
+          const SizedBox(
+            width: 20.0,
+          ),
+          Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                title,
+                style: theme.typography.heading7
+                    .copyWith(fontWeight: FontWeight.w600),
```

Comment:
I guess, You are also confused by Figma's UI just like me a few months back.
Figma shows all the properties of the style for on component along with their style token name. 

Style token size: 
<img width="671" alt="Screenshot 2024-04-07 at 2 16 27 PM" src="https://github.com/tarkalabs/tarka-ui-kit-flutter/assets/5791518/b4cc8c58-d174-4ae7-913a-652ce550d1cc">


Style size in component:
<img width="671" alt="Screenshot 2024-04-07 at 2 16 54 PM" src="https://github.com/tarkalabs/tarka-ui-kit-flutter/assets/5791518/f6949c91-aeb0-4038-91a5-3df627677456">

You can see font-weight is the same as what was given to the base token `headline7`.
We just need to specify `headline7` as the font style and do not need to override font-weight. 
Hope Figma makes their UI clear in the future 🤞

Reply:
Awesome thanks for sharing the observation :).